src/MultiThreadedCaches.jl

```diff
@@ -151,10 +185,10 @@ function Base.get!(func::Base.Callable, cache::MultiThreadedCache{K,V}, key) whe
         Base.@lock tlock begin
             if test_haskey
                 if !haskey(tcache, key)
-                    setindex!(tcache, key, v)
+                    tcache[key] = v
                 end
             else
-                setindex!(tcache, key, v)
+                tcache[key] = v
```

Comment:
😅 looks like we don't have good test coverage.. the key and value were backwards in these calls. i always screw that up with setindex 😅

Reply:
Oh, haha, it's because i was using `Dict{Int,Int}` everywhere in the tests so i never noticed 😂 😭 😂

Reply:
K, added a test for this in 073cbc5.